dataland-frontend/src/utils/DataPoint.ts

```diff
@@ -0,0 +1,24 @@
+import { type BaseDocumentReference, type ExtendedDocumentReference, type QualityOptions } from "@clients/backend";
+
+export interface GenericDataPoint<T> {
```

Comment:
Why does the generic data point already have a currency?

Reply:
On it

Reply:
It does not seem like this was necessary. However, the reason for having these generic datapoints is that we do not have something like them generated from the backend open api docs. But they are quite useful when specifying parameter and return types of functions and prevent us from listing every specific datapoint type.